src/main/java/tds/exam/results/services/impl/ExamResultsServiceImpl.java

```diff
@@ -65,20 +69,26 @@ public TDSReport findAndSendExamResults(final UUID examId) {
                 findAndSendExamResults(examId, report);
             } catch (Exception e) {
                 // Log error, and do not rethrow to prevent ERT from re-processing this same request
-                log.error("Error occurred while processing or sending the exam results for examId {}: " + e.getStackTrace());
+                log.error("Error occurred while processing or sending the exam results for examId {}: {}", examId,
+                    e.getMessage());
             }
         }
 
         return report;
     }
 
     private void findAndSendExamResults(final UUID examId, final TDSReport report) {
+        //TODO: Look into making these service calls asynchronously
```

Comment:
Rather than create a TODO can we create a JIRA?  I also don't think this will ever be the bottleneck performance wise so I'm not sure there is a lot of gain by doing async.  Agree they could be but I prefer to only add threading when needed.